examples/task-scheduler-timer/task_scheduler_timer.bal

```diff
@@ -27,7 +27,7 @@ public function main() returns error? {
     // Starts the timer.
     check timer.start();
 
-    runtime:sleep(4500);
+    runtime:sleep(4.5);
```

Comment:
Will this fix the intermittent hanging issue of this BBE test?

Reply:
Yes. Earlier the sleep interval was in milliseconds but now, it is in seconds.

Reply:
That wasn't the issue in the test hanging. Check this error log:

https://github.com/ballerina-platform/ballerina-distribution/runs/1719729266?check_suite_focus=true

Reply:
Oh, I may be wrong. Let's merge this.